cms/djangoapps/contentstore/courseware_index.py

```diff
@@ -188,6 +188,12 @@ def _track_index_request(cls, event_name, indexed_count):
             'category': 'courseware_index',
         }
 
+        self._emit_track_event(event_name, data)
```

Comment:
@dino-cikatic - I'd undo this change - unfortunately after merging, this change is no longer correct. I think it is actually unnecessary.